src/object/minishSizedArchway.c

```diff
@@ -0,0 +1,18 @@
+#include "global.h"
+#include "entity.h"
+#include "sprite.h"
+
+extern u32 CheckIsDungeon();
+
+void MinishSizedArchway(Entity* this) {
+    if (this->action == 0) {
+        this->action = 1;
+        this->frameIndex = this->type;
+        this->collisionLayer = 2;
+        UpdateSpriteForCollisionLayer(this);
+        if (CheckIsDungeon()) {
+            this->spritePriority.b0 = (this->spritePriority.b0 & 0xf8) | 1;
```

Comment:
does this match if you do `this->spritePriority.bo |= 1`?

Reply:
that's not even functionally equivalent? or do you mean to split it into two lines?

Reply:
```suggestion
            this->spritePriority.b0 &= 0xf8;
            this->spritePriority.b0 |= 1;
``` does match, yeah.

Reply:
Ah you're right. Perhaps it should be on two lines if that is possible.